packages/escape-html/src/index.js

```diff
@@ -51,6 +51,17 @@ export function escapeLessThan( value ) {
 	return value.replace( /</g, '&lt;' );
 }
 
+/**
+ * Returns a string with greater-than sign replaced.
+ *
+ * @param {string} value Original string.
+ *
+ * @return {string} Escaped string.
+ */
+export function escapeGreaterThan( value ) {
```

Comment:
I think it would be best if we avoided exposing this on the public interface of the module, to avoid committing ourselves to future maintenance if it's arguably a workaround for a WordPress quirk.

Noting that it's exported for use by testing, alternatives could include one of either:

- Moving it to a separate file from which it is the default export, but is imported and then not subsequently exported by `index.js`
- Naming it as an [unstable API](https://github.com/WordPress/gutenberg/blob/83f03b18ee0cc29d3c309da2c5d9a39392eefa75/docs/contributors/coding-guidelines.md#experimental-apis) `__unstableEscapeGreaterThan`